assets/js/index.js

```diff
@@ -0,0 +1,249 @@
+         
+          <div class="col-sm-12">
+            
+          </div>
+        </div>
+`);
+};
+
+const renderForecastWeather = (forecastWeatherData) => {
+  // render the forecast weather data and append each card to section
+  $("#forecastWeather").append(`  <div class="card" style="width: 10rem">
+  <img src="./assets/images/png/${icon}.png" class="card-img-top rounded float-start" alt="..." />
+  <div class="card-body">
+    <h5 class="card-title title">${date}</h5>
+    <p class="weatherDescription">Temp:${temp}°C</p>
+   </div>
+</div>`);
+};
+
+const renderWeatherData = (search) => {
+  const currentWeatherUrl = `https://api.openweathermap.org/data/2.5/weather?q=${search}&units=metric&appid=${API_KEY}`;
+  console.log(currentWeatherUrl);
+  let items = [];
+  // from the response cherry pick all the data you want to see in the current weather card
+  fetch(currentWeatherUrl)
+    .then(function (res) {
+      return res.json();
+    })
+    .then(function (result) {
+      items = result;
+      console.log(items);
+      lat = items.coord.lat;
+      lon = items.coord.lon;
+      renderForecastWeatherData(lat, lon);
+      city = items.name;
+      temp = items.main.temp;
+      icon = items.weather[0].icon;
+      console.log(icon);
+      wind = items.wind.speed;
+      humidity = items.main.humidity;
+      tempDate = items.dt;
+      date = moment.unix(tempDate).format("DD-MM-YYYY");
+      uv = description = items.weather[0].description;
+      let currentWeatherData = [temp, icon, wind, humidity, description, date];
+      renderCurrentWeather(currentWeatherData);
+    });
+
+  // render current weather data
+};
+const renderForecastWeatherData = (lat, lon) => {
+  // get the lat and lon from current weather data API response
+  const forecastWeatherUrl = `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=current,minutely,hourly&units=metric&appid=${API_KEY}`;
+  console.log(forecastWeatherUrl);
+  // from the response cherry pick all the data you want to see in the current weather card
+  let items = [];
+  // from the response cherry pick all the data you want to see in the current weather card
+  fetch(forecastWeatherUrl)
+    .then(function (res) {
+      return res.json();
+    })
+    .then(function (result) {
+      items = result;
+      console.log(items);
+
+      for (i = 1; i < 6; i++) {
+        temp = items.daily[i].temp.day;
+        icon = items.daily[i].weather[0].icon;
+        console.log(icon);
+        tempDate = items.daily[i].dt;
+        date = moment.unix(tempDate).format("ddd DD-MM");
+        let forecastWeatherData = [temp, icon, date];
+        renderForecastWeather(forecastWeatherData);
+      }
+    });
+  // render forecast weather data
+};
+const handleRecentSubmit = (event) => {
+  event.preventDefault();
+  //get text input
+  const container = document.getElementById("weatherContainer");
+  container.remove();
+  const main = document.getElementById("main");
```

Comment:
This variable is redundant since you are targeting the same thing on the line below using jquery